File: reports/Филатов Даниил Дмитриевич/5/src/task_2/src/com/company/Car.java
package com.company;
import java.math.BigDecimal;

public class Car extends PassengerCarrier {
    public Car(int speed, BigDecimal pricePerKm) {
        super(speed, pricePerKm);
    }
    void printTypeOfTransport() {
        System.out.println("Car:");
    }
    void calculateMovement(int distance) {
        System.out.println("A trip by car will take " + distance/this.getSpeed()
                + " hours and will cost " +
                this.getPricePerKm().multiply(BigDecimal.valueOf(distance)));
    }
}

File: reports/Loseiko/lab5/src/Car.cs
using System;

namespace lab5._2._9
{
    public class Car : Vehicle
    {
        public Car(int speed, int passengerTransportationCost, int сargoTransportationCost) : base(speed, passengerTransportationCost, сargoTransportationCost)
        {

        }
        public override void printTypeOfTransport()
        {
            Console.WriteLine("Машина: ");
        }
        public override void calculateMovement(int distance)
        {
            int hour = distance / GetSpeed();
            Console.WriteLine("Поездка на машине займет " + hour + " час(ов) и будет стоить " + GetPassengerTransportationCost() * hour + " рублей для пассажиров, и " + GetCargoTransportationCost() * hour + " рублей для товара ");
        }
    }
}
